src/locales/pl.po

```diff
@@ -3141,83 +3113,101 @@ msgstr "Osiągnie"
 
-msgid "asian up"
-msgstr "azjatyckie w górę"
+msgid "Win payout if %3 goes outside %7 and %6 through %5 after %4."
+msgstr "Wypłata nastąpi, pod warunkiem że %3 przekroczy %7 i %6 w przeciągu: %5 po %4."
 
-msgid "Win payout if the last tick of %3 is strictly higher than the average of the %plural(%5,%d tick,%d ticks)."
-msgstr "Wypłata nastąpi, pod warunkiem że ostatnia najmniejsza zmiana ceny %3 będzie dokładnie wyższa niż średnia %plural(%5,%d najmniejszej zmiany ceny,%d najmniejszych zmian ceny)."
+msgid "ends between"
+msgstr "kończy się pomiędzy"
+
+msgid "Win payout if %3 ends strictly between %7 to %6 at %5."
+msgstr "Wypłata nastąpi, pod warunkiem że %3 zakończy się dokładnie pomiędzy %7 a %6: %5."
+
+msgid "Win payout if %3 ends strictly between %7 to %6 at %5 after %4."
+msgstr "Wypłata nastąpi, pod warunkiem że %3 zakończy się dokładnie pomiędzy %7 a %6: %5 po %4."
+
+msgid "spread down"
+msgstr "spread w dół"
+
+msgid "%1 %2 payout for every point %3 falls from entry level, with stop loss of %6 %4 and stop profit of %6 %5."
+msgstr "Wypłata %2 %1 dla każdego punktu %3 spadła z poziomu wejściowego z limitem stop stratom wynoszącym %6 %4 i limitem stop zyskom wynoszącym %6 %5."
+
+msgid "%1 %2 payout for every point %3 falls from entry level, with stop loss of %4 points and stop profit of %5 points."
+msgstr "Wypłata %2 %1 dla każdego punktu %3 spadła z poziomu wejściowego z limitem stop stratom wynoszącym %4 punkty/punktów i limitem stop zyskom wynoszącym %5 punkty/punktów."
+
+msgid "touches"
+msgstr "osiąga"
+
+msgid "Win payout if %3 touches %6 through %5."
+msgstr "Wypłata nastąpi, pod warunkiem że %3 osiągnie %6 w przeciągu: %5."
+
+msgid "Win payout if %3 touches %6 through %5 after %4."
+msgstr "Wypłata nastąpi, pod warunkiem że %3 osiągnie %6 w przeciągu: %5 po %4."
 
 msgid "higher"
 msgstr "wyższa"
 
-msgid "Win payout if %3 is strictly higher than %6 at %5."
-msgstr "Wypłata nastąpi, pod warunkiem że %3 będzie dokładnie wyższy niż %6: %5."
+msgid "Win payout if %3 is higher than or equal to %6 at %5."
+msgstr "Wypłata nastąpi, pod warunkiem że %3 jest wyższy lub równy %6: %5."
 
-msgid "Win payout if %3 is strictly higher than %6 at %5 after %4."
-msgstr "Wypłata nastąpi pod warunkiem, że %3 będzie dokładnie wyższy niż %6: %5 po %4."
+msgid "Win payout if %3 is higher than or equal to %6 at %5 after %4."
+msgstr "Wypłata nastąpi pod warunkiem, że %3 jest wyższy lub równy %6: %5 po %4."
 
-msgid "Win payout if %3 after %plural(%5,%d tick,%d ticks) is strictly higher than %6."
-msgstr "Wypłata nastąpi pod warunkiem, że %3 po %plural(%5,%d najmniejsza zmiana ceny,%d najmniejsze zmiany ceny) będzie dokładnie wyższy niż %6."
+msgid "Win payout if %3 after %plural(%5,%d tick,%d ticks) is higher than or equal to %6."
+msgstr "Wypłata nastąpi pod warunkiem, że %3 po %plural(%5,%d najmniejsza zmiana ceny,%d najmniejsze zmiany ceny) jest wyższy lub równy %6."
```

Comment:
This is also buggy. Polish has 3 plural forms. The translator has provided only 2.